README.adoc

```diff
@@ -279,16 +280,16 @@ The Cloud Spanner Hibernate Dialect supports most of the standard Hibernate and
 [options="header"]
 |===
 | Unsupported Feature | Description
-| Large DML Transactions | Each Spanner transaction may only have up to 20,000 operations which modify rows of a table.
+| Large DML Transaction Limits | Each Spanner transaction may only have up to 20,000 operations which modify rows of a table.
```

Comment:
This should probably stay as "Transaction", since the column header is "unsupported feature".

Reply:
Done.